template/default.tmpl

```diff
@@ -197,3 +197,10 @@ Alerts Resolved:
 {{- end -}}
 {{- $priority -}}
 {{- end -}}
+
+{{ define "rocketchat.default.title" }}{{ template "__subject" . }}{{ end }}
+{{ define "rocketchat.default.alias" }}{{ template "__alertmanager" . }}{{ end }}
+{{ define "rocketchat.default.titlelink" }}{{ template "__alertmanagerURL" . }}{{ end }}
+{{ define "rocketchat.default.emoji" }}{{ end }}
+{{ define "rocketchat.default.iconurl" }}{{ end }}
+{{ define "rocketchat.default.text" }}{{ end }}
```

Comment:
If the default text template has no content, what does a rocket message look like without customization? Can you share a screenshot perhaps?

Reply:
This it what it looks like on mobile
![image](https://github.com/user-attachments/assets/84cf3b5c-68d9-4eb7-a3b1-62f545db264a)

Reply:
Just tested it in Slack and it's the same! 👍